docs/src/examples/components/Chat/Content/ChatExampleReactionGroup.shorthand.tsx

```diff
@@ -0,0 +1,58 @@
+import { Avatar, Chat } from '@stardust-ui/react'
+import * as React from 'react'
+
+const reactions = [{ icon: 'thumbs up', content: 5 }, { icon: 'thumbs down', content: '1K' }]
+
+const items = [
+  {
+    attached: 'top',
+    contentPosition: 'end',
+    message: {
+      content: (
+        <Chat.Message
+          reactionGroup={{ items: reactions }}
+          content="Hello"
+          author="John Doe"
+          timestamp="Yesterday, 10:15 PM"
+          mine
+        />
+      ),
+    },
+    key: 'message-1',
+  },
+  {
+    attached: 'bottom',
+    contentPosition: 'end',
+    key: 'message-2',
+    message: {
+      content: (
+        <Chat.Message
+          reactionGroup={{ items: [{ icon: 'thumbs up', content: '8' }] }}
```

Comment:
just as a matter of thought - here, also if I would be a client, would expect the following syntax being possible as well, without any syntax noise `items` prop is currently introducing:

```jsx
<Chat.Message
  reactions={[ 
     { icon: ...},
     { ... }
  ]}
/>
```

As a proof for this desire we might take an initial state of `Reactions` proposal described in the corresponding RFC: https://github.com/stardust-ui/react/issues/942 .

Should note that this issue applies not just to this case exclusively - thus, we might want to think about a general solution, as this is, essentially, about adjusting the logic of how shorthand values are interpreted

Reply:
Actually, it's the same issue as `actionMenu` 💭

Reply:
Please see my comment: https://github.com/stardust-ui/react/pull/959#discussion_r260761334 Generally at this moment I don't have better proposal, so would like to preserve consistency, so that when we start fixing this we can fix all occurrences and again will have consistency around these collection components